src/root/components/CountryMatches.js

```diff
@@ -0,0 +1,116 @@
+export const CountryMatches = {
```

Comment:
@szabozoltan69 sorry am just looking at this, but can we please please try and avoid hard-coding this list like this? We should just be able to get the countries from the redux store and then loop through them to search by substring?

Reply:
@batpad We can find an other solution, yes, in the next release :-) – but then we should implement some logic also with it, which is not straightforward. E.g. the 'United Arab Emirates' can be found now with 'un', 'ar', or with 'em'. But this rule (beginning of words' is not always works, because there are a lot of "Republic..."-s or "Kingdom..."-s).
And there are other tricks like this, e.g. "us" -> United States, "uk" -> United Kingdom etc.
So, for now, it seemed to be the more simple way to create this collection to have a relevant search possibility.

My hardest question is to have a better solution instead of this: 2 letters -> wired in list, 3 letters -> ElasticSearch.
Instead of we would like to know that "the answer still not arrived, in the meantime we show this".